server/settings.go

```diff
@@ -10,20 +10,17 @@ const (
 	settingOff = "off"
 )
 
+var onOffToBool = map[string]bool{settingOn: true, settingOff: false}
```

Comment:
nit: 0/5 an (inline-able) function with a switch statement is slightly preferred. Better performance (not that it matters here) and a tad easier on the eye?